lib/clustering/ClusterRPC.ts

```diff
@@ -74,6 +74,7 @@ const rpcLogger = new werelogs.Logger('ClusterRPC');
 
 export type ResultObject = {
     error: Error | null;
+    errorCode?: number;
```

Comment:
I'd prefer having the error code part of the `error` object itself to keep it tidy, where the `Error` can be reconstructed from something serialized. I'm pretty sure the `Error` type accepts a `code` attribute. The current serialization seems to be a simple string now but it could be instead an object with two fields for example (say `code` and `message`) if the goal is to keep the description of the error along with the code.

The serialization/deserialization should stay internal to ClusterRPC and not exposed to clients so it should be fine to change how it's made.

Reply:
I moved the code to the `error`